src/status_im/ui2/screens/chat/messages/message.cljs

```diff
@@ -40,7 +40,8 @@
   (rf/sub [:contacts/contact-name-by-identity from]))
 
 (defn render-inline
-  [_message-text content-type acc {:keys [type literal destination]}]
+  [_message-text content-type acc {:keys [type literal destination]} use-status-tag?
+   community-channel-names-and-ids community-id current-chat-id]
```

Comment:
I think we should avoid passing so much state to a message for now, as this might cause re-rendering.

a bit less correct, but simpler for now, is to pass "community-id" only, and just on-press fire that one with the literal, and do the work on the event handler instead.

so in this case:

```
[_message-text content-type acc {:keys [type literal destination]} use-status-tag? community-id current-chat-id]
   (when use-status-tag?
               {:style    {:color                :blue
                           :text-decoration-line :underline}
                :on-press #(rf/dispatch [:communities/status-tag-pressed community-id literal]))})

```

And in the event handler, you do the lookup, and navigate to chat if a chat is found.

This is a bit less nice in terms of UX, since  any link will show as blue, even if there's not chat with that name,  but it's going to be more performant & simpler, I think that's an acceptable compromise.